integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/context/resolver/WebhookSiteTokenResolver.java

```diff
@@ -0,0 +1,24 @@
+package com.redhat.service.smartevents.integration.tests.context.resolver;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import com.redhat.service.smartevents.integration.tests.context.TestContext;
+import com.redhat.service.smartevents.integration.tests.resources.webhook.site.WebhookSiteAccount;
+
+public class WebhookSiteTokenResolver implements Resolver {
+
+    private static final Pattern WEBHOOK_SITE_TOKEN_REGEX = Pattern.compile("\\$\\{webhook\\.site\\.token\\.([^\\}]+)\\}");
+
+    public boolean match(String placeholder) {
+        return WEBHOOK_SITE_TOKEN_REGEX.matcher(placeholder).find();
+    }
+
+    public String replace(String content, TestContext context) {
+        Matcher matcher = WEBHOOK_SITE_TOKEN_REGEX.matcher(content);
+        return matcher.replaceAll(match -> {
+            String tokenName = match.group(1);
+            return WebhookSiteAccount.getToken(tokenName);
```

Comment:
Will the "getToken" method trigger the method "Accounts.get(WebhookAccount.class).token(tokenName)" and probably some REST call every time the token is used or is it somehow cached in TNB?

Reply:
>Will the "getToken" method trigger the method "Accounts.get(WebhookAccount.class).token(tokenName)"
yes

>probably some REST call every time the token is used or is it somehow cached in TNB?
no, the token is retrieved from localconfig.yaml, there is no REST call when invoking this method